src/models/credentials/write.js

```diff
@@ -62,21 +62,26 @@ export const setVerificationRejected = async (id) => {
   }
 }
 
-export const requestExperience = async (id, userId, orgId, message, exact_info) => {
+export const requestExperience = async (id, userId, orgId, message, exact_info, options = {}) => {
+
+  const {issuedByOrg} = options;
+
   try {
     const { rows } = await app.db.query(sql`
       INSERT INTO experience_credentials (
         user_id,
         org_id,
         experience_id,
         message,
-        exact_info
+        exact_info,
+        is_issued
```

Comment:
No, issued should be status of `experience_credentials`

Reply:
You mean rename the `is_issued` to `issued`?

Reply:
It's resolved i think